cfe_internal/enterprise/templates/httpd.conf.mustache

```diff
@@ -194,6 +194,9 @@ LogLevel warn
     SSLEngine on
     SSLCertificateFile "{{{vars.cfe_internal_hub_vars.SSLCertificateFile}}}"
     SSLCertificateKeyFile "{{{vars.cfe_internal_hub_vars.SSLCertificateKeyFile}}}"
+{{^classes.cfe_enterprise_have_SSLCACertificateFile_defined}}
+    SSLCACertificateFile "{{{vars.cfe_internal_hub_vars.SSLCACertificateFile}}}"
+{{/classes.cfe_enterprise_have_SSLCACertificateFile_defined}}
```

Comment:
I was hitting an issue with this breaking my `httpd.conf` with `SSLCACertificateFile ""` while it cannot be an empty value. I didn't define the `SSLCACertificateFile` policy variable anywhere.